scripts/setup/configure-certs

```diff
@@ -0,0 +1,24 @@
+#!/bin/env bash
+
+# Realm-specific construct
+if [ -z "$RLMNAME" ]; then
+    echo "No Realm name given."
+    RLMNAME="A boson of realm"
+fi
+
+echo "Executing certificates configuration..."
+if [ ! -e /etc/ssl/private/zulip.key ] && [ ! -e /etc/ssl/certs/zulip.combined-chain.crt ]; then
+    echo "SSL certificates for Zulip not found in /etc/ssl/."
+    echo "Autogenerating certificates ..."
+    sudo -i
+    apt-get install -y openssl
+    openssl genrsa -des3 -passout pass:x -out /tmp/server.pass.key 4096
+    openssl rsa -passin pass:x -in /tmp/server.pass.key -out /etc/ssl/private/zulip.key
+    openssl req -new -nodes -subj "/O=$RLMNAME" -key /etc/ssl/private/zulip.key -out /tmp/server.csr
+    openssl x509 -req -days 365 -in /tmp/server.csr -signkey /etc/ssl/private/zulip.key -out /etc/ssl/certs/zulip.combined-chain.crt
+    rm -f /tmp/server.csr /tmp/server.pass.key
+    echo "SSL certificates autogeneration succeeded."
+else
+    echo "SSL certificates for Zulip already exist in /etc/ssl/. Continuing."
+fi
+echo "Certificates configuration succeeded."
```

Comment:
This script, with a bit of cleanup, could be used to simplify our documentation for SSL certificates (we can just recommend the script for using self-signed).